gpib_server.py

```diff
@@ -194,6 +195,20 @@ def query(self, c, data):
         ans = instr.read_raw()
         return str(ans).strip()
 
+    @setting(7, n_bytes='w', returns='s')
+    def read_raw(self, c, n_bytes=None):
+        """Read from the GPIB bus.  No .strip()
```

Comment:
Saying "No .strip()" will probably be confusing to clients looking at the labrad setting docs who don't have the source code in front of them and don't know the history of these changes. I would suggest saying something like:
```
"""Read raw bytes from the GPIB bus.

Termination characters, if any, will not be stripped.
If n_bytes is specified, reads only that many bytes.
Otherwise, reads until the device stops sending.
```

Then, we should modify the `read` setting docs to indicate that it _does_ strip termination characters.